src/root/views/region.js

```diff
@@ -77,25 +77,26 @@ class AdminArea extends SFPComponent {
     this.onAdditionalLinkClickAction = this.onAdditionalLinkClickAction.bind(this);
   }
 
-  getTabDetails () {
+  getTabDetails() {
     const { strings } = this.context;
     return [
       { title: strings.regionOperationsTab, hash: '#operations' },
       { title: strings.region3WTab, hash: '#3w' },
       { title: strings.regionRiskTab, hash: '#risk-watch' },
       { title: strings.regionProfileTab, hash: '#regional-profile' },
+      { title: strings.regionCountryPlansTab, hash: '#country-plans' },
```

Comment:
We should remove this as well. Also, this file only contains the white-space changes (probably auto-formatted during the save), please be careful of such changes.